funcx_web_service/routes/funcx.py

```diff
@@ -248,10 +227,16 @@ def submit(user: User):
     final_http_status = 200
     success_count = 0
     for task in tasks:
-        res = auth_and_launch(
-            user_id, function_uuid=task[0], endpoint_uuid=task[1],
-            input_data=task[2], app=app, token=token, task_group_id=task_group_id, serialize=serialize
-        )
+        task_uuid = str(uuid.uuid4())
+        try:
+            res = auth_and_launch(
```

Comment:
`auth_and_launch` is also used in the [automate api](https://github.com/funcx-faas/funcx-web-service/blob/main/funcx_web_service/routes/automate.py#L72).  Given the signature changed to raise an exception should that code also change?

Reply:
Good catch, I instead changed the signature back to the original and moved the try block into the method itself so we don't have to wrap it outside